packages/apollo-server-integration-testsuite/src/ApolloServer.ts

```diff
@@ -261,6 +359,52 @@ export function testApolloServer<AS extends ApolloServerBase>(
           expect(result.data).toEqual({ hello: 'hi' });
           expect(result.errors).toBeUndefined();
         });
+
+        it("accepts a gateway's schema and calls its executor", async () => {
+          const { gateway, triggers } = makeGatewayMock();
+
+          const executor = jest.fn();
+          executor.mockReturnValue(
+            Promise.resolve({ data: { testString: 'hi - but federated!' } }),
+          );
+
+          triggers.resolveLoad({ schema, executor });
+
+          const { url: uri } = await createApolloServer({ gateway });
+
+          const apolloFetch = createApolloFetch({ uri });
+          const result = await apolloFetch({ query: '{testString}' });
+
+          expect(result.data).toEqual({ testString: 'hi - but federated!' });
+          expect(result.errors).toBeUndefined();
+          expect(executor).toHaveBeenCalled();
+        });
+
+        // Mocks are not used when operating as a gateway. Todo?
+        it.skip("allows mocking a gateway's operations", async () => {
```

Comment:
This `it.skip`s have had a tendency to bit rot, historically, so unless this is on the short-term `TODO` list, I'd suggest we remove it.

Reply:
Just wanted to make it clear that this wasn't working. If that's find we can go ahead and remove it.